split_train_valid.py

```diff
@@ -105,10 +112,13 @@ def main(args):
         )
 
     else:
+        # p is the approximation of percent of dataset to split
+        p = args.percent_split/100
         print('Loading split from csv...')
         visitors_df = pd.read_csv(
             os.path.join(args.data_dir, args.split_file),
             dtype=str,
+            skiprows=lambda i: i>0 and random.random() > p
```

Comment:
I'd leave this change out, because it will behave weirdly if you run this script multiple times. Right now, if you leave the --fresh_split option off, this will select p% of the users from the original split file, overwrite the file to have p% of the users in the split, and continue. If you run it again with the same options, it will select p% of those users and overwrite the split again, making the dataset even smaller. Intuitively I wouldn't expect the `--fresh_split=false` option to create a new random split, but that's what this will do.

Reply:
To clarify, I'm just referring to this one line, I think the changes where `fresh_split=true` make sense.

Reply:
I got rid of the split when fresh_split=false